lib/secure_headers.rb

```diff
@@ -163,7 +163,7 @@ def opt_out_of_all_protection(request)
     # returned is meant to be merged into the header value from `@app.call(env)`
     # in Rack middleware.
     def header_hash_for(request)
-      config = config_for(request, prevent_dup = true)
+      config = config_for(request, _prevent_dup = true)
```

Comment:
🤔 an early mentor of mine emphasized not passing around "magic booleans" (or any magic things really). I guess we could add a second method or treat the value as an options hash.

Maybe in 4.0 we can drop < 2.1 and use kwargs.